### DISCUSSION TITLE
How to save the prompt

### DISCUSSION THREAD

**Discussion by junle-chen** [sinaptik-ai/pandas-ai#1058] [Q&A] [answered]
How to save the prompt in pandasai. For example, I want him to not need to sort when grouping queries. I can pre-define the prompt so that the next time I open a new session, I don't need to add a prompt. He can not sort by default when grouping and sorting to improve efficiency.

**Comment by gventuri** [comment, marked as answer]
@HITjunle the prompt is a string, so you can save as part of your code and just pass it to the SmartDataframe or the Agent.
I'm not 100% I understand your need, but if the prompt is static, you can just save it in your python file.
